release_draft.md

```diff
@@ -0,0 +1,303 @@
+      1. `ign-fooX-debbuilder` job built the packages and pass them to the
+    `repository_uploader_packages` job.
+      1. `generic-release-homebrew_pull_request_updater` will open a
+         new PR to coordinate the release process in `homebrew-simulation`.
+ 1. `repository_uploader_packages` import the packages created by the
+    `ign-fooX-debbuilder` job (there will be one build of the job for each
+    platform combination of Ubuntu/Debian release + architecture) and upload the
+    .deb packages to `packages.osrfoundation.org` and `osrf-distributions S3`.
+ 1. For macOS the PR in `homebrew-simulation` waits for a comment from an
+    Ignition developer with the order `build bottle` that will trigger the job
+    `generic-release-homebrew_triggered_bottle_builder`.
+ 1. `generic-release-homebrew_triggered_bottle_builder`will use the tarball with
+    release sources from `osrf-distributions S3` to generate the binary bottles.
+    They will be uploaded to `osrf-distributions S3`.
+
+### Prerequisites
+
+To perform a new release a small number of configurations and credentials are
+needed for the developer system that will trigger it. If a permanent operative
+system is used for releasing these installation steps needs to be executed once.
+
+#### Team and development checks
+
+When creating a new release there are some guidelines to follow before starting
+the process:
+
+ * Asking the team if there are any concerns about making the release
+ * Check if there are changes to previous versions that need to be forward-ported.
+ * See if there are open PRs against that release branch (release branch is the
+   one with the name `ign-fooX` where foo is the name of the Ignition library and
+   X the major version of the version bump planned) that could go into the new
+   release.
+
+#### Software and configurations
+
+The releasing process is only supported in Linux at this moment.
+
+The tool designed to facilitate the process of releasing software for all
+platforms is called `release.py` and is hosted in the
+[release-tools](https://github.com/ignition-tooling/release-tools/blob/master/release.py)
+repository. Installing a copy of the repository is required to perform a new
+release:
+
+```
+git clone https://github.com/ignition-tooling/release-tools.git
+```
+
+The `release.py` tool will use `sc3md` software to interact with Amazon AWS
+storage to host sources and binaries.
+
+```
+sudo apt-get install s3cmd
+```
+
+Some Debian tools require the following variables to be set:
+```bash
+export DEBEMAIL="<username>@openrobotics"
+export DEBFULLNAME="<Your full name>"
+```
+**Note:** the two above exported variables can be added to `.bashrc` to have
+them configured automatically on every run.
+
+
+#### Credentials
+
+There is a small number of credentials needed to interact with the release
+process:
+
+ * S3 access to open robotics:
+  ```
+  s3cmd --configure
+  ```
+  If you don't have AWS credentials please contact with TODO
```

Comment:
@j-rivero we'll get some internal documentation on how to set up accounts for folks.
In the meantime are you okay being mentioned like this in public documentation or do you have a suggested alternative?

```suggestion
  If you don't have AWS credentials please contact @j-rivero or @nuclearsandwich who will help set you up.
```

Reply:
> @j-rivero we'll get some internal documentation on how to set up accounts for folks.
> In the meantime are you okay being mentioned like this in public documentation or do you have a suggested alternative?

sure no problem. We can start with this and see how it works, I don't expect too much activity with AWS credentials so anything simple sound good.to me.